recipes/orc/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.5.3" %}
+
+package:
+  name: orc
+  version: {{ version }}
+
+source:
+  url: https://archive.apache.org/dist/orc/orc-{{ version }}/orc-{{ version }}.tar.gz
+  sha256: 96da3cccd2b396dc7813949d3c459a5d6995778928d344f4553cf890f171521c
+  patches:
+    - 0001-Deactivate-aggressive-failures-on-warnings.patch
```

Comment:
Please, add some explanation why this patch is necessary

Reply:
Done